Jobs/test/Job.IdentityDeletion.Tests/Tests/ActualDeletionWorkerTests.cs

```diff
@@ -103,4 +103,9 @@ private void SetupRipeDeletionProcessesCommand(IMediator mediator, params Identi
         var logger = A.Dummy<ILogger<ActualDeletionWorker>>();
         return new ActualDeletionWorker(hostApplicationLifetime, identityDeleters, mediator, pushNotificationSender, eventBus, logger);
     }
+
+    private static ActualDeletionWorker CreateWorker(IMediator mediator, IPushNotificationSender pushNotificationSender)
```

Comment:
Methods should be placed below the methods that use them.

So this one should be placed above the one with 4 parameters